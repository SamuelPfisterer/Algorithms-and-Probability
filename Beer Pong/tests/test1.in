30
2 4 19 1
0.98077 0.701338
2 9 19 1
0.822325 0.796975
2 10 2 1
0.515992 0.502784
2 8 20 1
0.916676 0.626038
2 12 17 1
0.64873 0.779309
2 3 18 1
0.871184 0.695318
2 18 7 1
0.913636 0.840415
2 11 11 1
0.585118 0.625029
2 11 6 1
0.752414 0.593647
2 10 15 1
0.764296 0.964008
2 7 18 1
0.665924 0.851403
2 20 6 1
0.855867 0.946539
2 16 4 1
0.705886 0.997429
2 7 20 1
0.951802 0.643403
2 3 8 1
0.804071 0.943161
2 11 12 1
0.476151 0.518612
2 10 16 1
0.608568 0.995208
2 3 8 1
0.554024 0.750216
2 15 9 1
0.896841 0.547791
2 12 9 1
0.700792 0.913246
2 2 15 1
0.525268 0.750351
2 20 9 1
0.906089 0.985638
2 4 19 1
0.877605 0.567281
2 19 4 1
0.398402 0.900859
2 16 19 1
0.898021 0.840115
2 5 3 1
0.649502 0.548223
2 20 13 1
0.612655 0.559561
2 9 12 1
0.363016 0.675368
2 13 19 1
0.85759 0.827087
2 12 11 1
0.913574 0.956759
