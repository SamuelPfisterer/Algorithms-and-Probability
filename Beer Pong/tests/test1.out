0.8102013
0.5580062
0.0859987
0.6547686
0.38029
0.7467291
0.2558181
0.292559
0.2655579
0.4585776
0.4794653
0.1975078
0.1411772
0.7050385
0.5847789
0.2484266
0.3745034
0.4029265
0.3363154
0.3003394
0.4634718
0.2812
0.724978
0.0692873
0.4874971
0.2435633
0.2413489
0.2074377
0.5091941
0.4369267
